packages/feedback/src/core/sendFeedback.ts

```diff
@@ -13,16 +13,17 @@ export const sendFeedback: SendFeedback = (
   hint: EventHint & { includeReplay?: boolean } = { includeReplay: true },
 ): Promise<string> => {
   if (!options.message) {
-    throw new Error('Unable to submit feedback with empty message');
+    return Promise.reject(new Error('Unable to submit feedback with empty message.'));
```

Comment:
change here to reject with an Error object instead of throwing it. This shouldn't cause any problems for folks who were already using the `await` keyword to call this function because `await sendFeedback(...)`.

However this should improve things for folks who expect to do `sendFeedback(...).then(...)` because previously the return type was not a promise, so extra care would've needed to be taken.